composer.json

```diff
@@ -17,9 +17,9 @@
         }
     ],
     "require": {
-        "php":             ">=5.3.2",
+        "php":             "^5.4",
```

Comment:
We don't know if this is PHP7 compatible for now.

If yes, we could add this version.